todolist.js

```diff
@@ -0,0 +1,220 @@
+let tasksList = [];
+
+if (localStorage.getItem('tasks')) {
+  tasksList = JSON.parse(localStorage.getItem('tasks'));
```

Comment:
- I see that you are creating variables and conditions in the global scope. this is error-prone, kindly encloses it in a function instead